File: 58. Length of Last Word/58. Length of Last Word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int length = 0;
        int i = s.Length - 1;
        
        // Skip trailing spaces
        while (i >= 0 && s[i] == ' ') {
            i--;
        }
        
        // Count characters of the last word
        while (i >= 0 && s[i] != ' ') {
            length++;
            i--;
        }
        
        return length;
    }
}

File: 58. Length of Last Word/58. Length of Last Word.go
func lengthOfLastWord(s string) int {
    length := 0
    i := len(s) - 1
    
    // Skip trailing spaces
    for i >= 0 && s[i] == ' ' {
        i--
    }
    
    // Count characters of the last word
    for i >= 0 && s[i] != ' ' {
        length++
        i--
    }
    
    return length
}